src/main/java/org/mockito/verification/After.java

```diff
@@ -20,13 +22,25 @@
      * <p>
      * Typically, you won't use this class explicitly. Instead use timeout() method on Mockito class.
      * See javadoc for {@link VerificationWithTimeout}
+     * @deprecated Use {@link After#After(Duration, VerificationMode)} instead.
      */
+    @Deprecated
     public After(long delayMillis, VerificationMode verificationMode) {
-        this(10, delayMillis, verificationMode);
+      this(Duration.ofMillis(delayMillis), verificationMode);
```

Comment:
Nit: we indent with 4 spaces. I am surprised that spotless didn't format that, I will reconfigure the autoformatter to take care of that.

Reply:
Done.